exercises/A2-A3/1-error-propagating/src/main.rs

```diff
@@ -0,0 +1,49 @@
+// This is a very basic rust implementation that approximates the UNIX 'wc' program for text-files; for example, the file you are reading now
```

Comment:
this is the same file as `exercises/A2-A3/1-error-propagating/src/bin/wc.rs` ?! why?

Reply:
Forgot to git rm :)